### DISCUSSION TITLE
GigaGAN model for upscayl?

### DISCUSSION THREAD

**Discussion by lancerdoom1** [upscayl/upscayl#617] [Ideas]
Hi i was wondering if we can get GigaGAN model for Upscayl.
https://mingukkang.github.io/GigaGAN/

**Comment by NayamAmarshe** [comment]
We'll always be upgrading and updating. If GigaGAN seems like a good alternative to the Real-ESRGAN model, we'll move to it :)

**Comment by SRC267** [comment]
> Hi i was wondering if we can get GigaGAN model for Upscayl.
> https://mingukkang.github.io/GigaGAN/

Or this: https://blog.fal.ai/introducing-aurasr-an-open-reproduction-of-the-gigagan-upscaler-2/
